xml/ha_storage_protection.xml

```diff
@@ -842,22 +800,126 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
-    node needs to be fenced. </para>
+    node needs to be fenced.</para>
+  </sect1>
+
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting Up Diskless SBD</title>
+   <para>SBD can be operated in a diskless mode. In this mode, a watchdog device
+    will be used to reset the node in the following cases: if it loses quorum,
+    if any monitored daemon is lost and not recovered, or if Pacemaker decides
+    that the node requires fencing. Diskless SBD is basically based on
+    <quote>self-fencing</quote> of a node, depending on the status of the cluster,
+    the quorum and some reasonable assumptions.</para>
+    <important>
+     <title>Number of Cluster Nodes</title>
+       <para>
+         Do <emphasis>not</emphasis> use diskless SBD as fencing mechanism
+         for two-node clusters. Use it only in clusters with three or more
+         nodes. SBD in diskless mode cannot handle split brain
+         scenarios for two-node clusters.
+      </para>
+   </important>
+
+   <procedure xml:id="pro.ha.storage.protect.confdiskless">
+    <title>Configuring Diskless SBD</title>
+    <step>
+     <para>Open the file <filename>/etc/sysconfig/sbd</filename> and use
+      the following entries:</para>
+     <screen>SBD_PACEMAKER=yes
+SBD_STARTMODE=always
+SBD_DELAY_START=no
+SBD_WATCHDOG_DEV=/dev/watchdog
+SBD_WATCHDOG_TIMEOUT=5</screen>
+      <para>
+       The <varname>SBD_DEVICE</varname> entry is not needed as no shared
+       disk is used. When this parameter is missing, the <systemitem>sbd</systemitem>
+       service does not start any watcher process for SBD devices.
+      </para>
+    </step>
+    <step>
+     <para>On each node, enable the SBD service:</para>
+     <screen>&prompt.root;<command>systemctl</command> enable sbd</screen>
+     <para>It will be started together with the Corosync service whenever the Pacemaker
+      service is started.</para>
+    </step>
+    <step>
+     <para>Restart the cluster stack on each node:</para>
+     <screen>&prompt.root;<command>systemctl</command> stop pacemaker
+&prompt.root;<command>systemctl</command> start pacemaker</screen>
+     <para> This automatically triggers the start of the SBD daemon. </para>
+    </step>
+    <step>
+      <para>
+       Check if the parameter <parameter>have-watchdog=true</parameter> has
+       been automatically set:
+      </para>
+      <screen>&prompt.root;<command>crm</command> configure show | grep have-watchdog
+         have-watchdog=true</screen>
+    </step>
+    <step>
+     <para>Run <command>crm configure</command> and set the following cluster
+      properties on the &crmshell;:</para>
+<screen>&prompt.crm.conf;<command>property</command> stonith-enabled="true" <xref
+        linkend="co.ha.sbd.st.enabled" xrefstyle="select:label"/>
+&prompt.crm.conf;<command>property</command> stonith-watchdog-timeout=10 <co
+        xml:id="co.ha.sbd.diskless.watchdog-timeout"/></screen>
```

Comment:
Somehow in the PDF, this is shown to have the conflicting indexing number (1)  which belongs to the previous one.